app/tests/api/db/test_migrations.py

```diff
@@ -60,8 +57,6 @@ def test_db_schema_non_session(monkeypatch):
 
 
 def test_db_setup_via_alembic_migration(test_db_schema_non_session):
-    # Change directory location so the relative script_location in alembic config works.
-    os.chdir(Path(__file__).parent.parent)
```

Comment:
Why don't we want to test this?

Reply:
It was making the test print the path for some reason + removing it didn't break anything so I think it's not relevant (note: this was copied verbatim, might be an old alembic issue).

Reply:
No idea what was going on, tinkered with it and it stopped printing.

I don't think we need it in either case (it works both ways). The bit I copied from grabbed the config differently by some hacky file paths, so likely just irrelevant in this approach.

I did add a bit to the test to validate via log statements that the migration occurred.